addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightHandlerBase.java

```diff
@@ -193,6 +194,19 @@ public void handleCommandHelper(ChannelUID channelUID, Command command, String l
                     handleIncreaseDecreaseBrightnessCommand((IncreaseDecreaseType) command);
                 }
                 break;
+            case CHANNEL_COMMAND:
+                // logger.info(logInfo, command);
```

Comment:
Commented code should be either removed or not commented.

Reply:
Fixed.